content/Editors/Code/options.md

```diff
@@ -28,20 +28,34 @@ You can also open the Settings editor from the Command Palette (`Ctrl+Shift+P`)
 Read more about settings in VSCode in [VSCode documentation](https://code.visualstudio.com/docs/getstarted/settings)
 
 ---
+#### `FSharp.fsac.dotnetArgs`
 
-#### `FSharp.fsac.netCoreDllPath`
+Additional CLI arguments that will be provided to the dotnet runtime when running FSAC
 
-The path to the `fsautocomplete.dll`, useful for debugging a self-built fsac.
+**Type:** `string []`
 
-**Type:** `string`
+**Default:** `[ ]`
 
-**Default:** ` `
+> Note: to enable .Net 6 support for FSAC at the present time, you should set this option to specify the .net 6 runtime specifically:
```

Comment:
here's the important docs for those that want to use .net 6 with ionide right now (until we get a better mechanism)